### PR TITLE
Update job definitions for rafter

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Update job definitions for rafter:
  - split common config to templates
  - write job definitions for `minio-gateway` and `minio-gateway-migration` jobs

**Related issue(s)**
https://github.com/kyma-project/rafter/issues/5

### CODE REVIEW

File: prow/jobs/rafter/rafter.yaml [hunk 1]
```diff
@@ -1,142 +1,279 @@
...
+
+common_job_config_template: &common_job_config_template
   decorate: true
-  optional: false
   path_alias: github.com/kyma-project/rafter
   max_concurrency: 10
-
-job_build_template: &job_build_template
-  <<: *common_config
   extra_refs:
```
Comment (aerfio) [inline prow/jobs/rafter/rafter.yaml]: Now that I think about it we should pin that repo, test-infra, to specific commit, what do you think?

Comment (magicmatatjahu) [reply prow/jobs/rafter/rafter.yaml]: I agree with you, but we must consider to which. I suggest to Friday last commit.

Comment (michal-hudy) [reply prow/jobs/rafter/rafter.yaml]: To latest available :)

File: prow/jobs/rafter/rafter.yaml [hunk 2]
```diff
@@ -1,142 +1,279 @@
...
 
-common_config: &common_config
-  skip_report: false
-  always_run: true
+common_job_labels_minio_az_gateway_template: &common_job_labels_minio_az_gateway_template
+  preset-rafter-minio-az-gateway: "true"
+  preset-creds-aks-kyma-integration: "true"
+
+additional_postsubmit_job_labels_template: &additional_postsubmit_job_labels_template
+  prow.kyma-project.io/slack.skipReport: "true"
```
Comment (michal-hudy) [inline prow/jobs/rafter/rafter.yaml]: I think we should enable reporting for all jobs, we are stable enough.

**Pull request merged**